index.html

```diff
@@ -10,7 +10,7 @@
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
     <script  src="js/index.js"></script>
 
-    <h2 id="title">Freecodecamp survey form</h2>
+    <h1 id="title">Freecodecamp survey form</h1>
```

Comment:
> ### In index.html
> Kindly make sure you add your title in a header tag. it will look nicer

:red_circle: I think there is no need to specify the file name in which the change should be made as this inline comment  has already showing us the file where change should occur. Keeping that line  means repeating yourself